contrib/macdeploy/macdeployqtplus

```diff
@@ -544,6 +544,20 @@ ds.close()
 if platform.system() == "Darwin":
     subprocess.check_call(f"codesign --deep --force --sign - {target}", shell=True)
 
+print("+ Installing background.tiff +")
+
+bg_path = os.path.join('dist', '.background', os.path.basename('background.tiff'))
+os.mkdir(os.path.dirname(bg_path))
+shutil.copy2('contrib/macdeploy/background.tiff', bg_path)
```

Comment:
Should probably also use `os.path.join` here to be consistent instead of assuming `/` seperator?

Reply:
Done.